packages/integration-karma/test/api/isNodeFromTemplate/index.spec.js

```diff
@@ -74,9 +72,7 @@ if (!process.env.NATIVE_SHADOW) {
         const span = document.createElement('span');
         elm.shadowRoot.querySelector('h2').appendChild(span);
 
-        return new Promise((resolve) => {
-            setTimeout(resolve);
-        }).then(() => {
+        return new Promise(setTimeout).then(() => {
```

Comment:
Nit, but I think there's [a good case](https://jakearchibald.com/2021/function-callback-risks/) for using the more verbose `new Promise(resolve => setTimeout(resolve))`